packages/flame/svg/Icons/qrcode.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+    <g id="qrcode">
+        <path id="base-2" opacity="0.7" d="m0,15l0,-2a1,1 0 0 1 2,0l0,1l1,0a1,1 0 0 1 0,2l-2,0a1,1 0 0 1 -1,-1zm16,-14l0,2a1,1 0 0 1 -2,0l0,-1l-1,0a1,1 0 0 1 0,-2l2,0a1,1 0 0 1 1,1zm-2,13l0,-1a1,1 0 0 1 2,0l0,2a1,1 0 0 1 -1,1l-2,0a1,1 0 0 1 0,-2l1,0zm-12,-12l0,1a1,1 0 1 1 -2,0l0,-2a1,1 0 0 1 1,-1l2,0a1,1 0 1 1 0,2l-1,0z"/>
+        <path id="base-1" d="m4.37907,0.80357zm3.21327,6.4061l0,-3.46912c0,-0.24559 -0.1842,-0.4298 -0.4298,-0.4298l-3.46912,0c-0.24559,0 -0.4298,0.1842 -0.4298,0.4298l0,3.46912c0,0.24559 0.1842,0.4298 0.4298,0.4298l3.46912,0c0.24559,0 0.4298,-0.1842 0.4298,-0.4298zm1.72943,-3.03932l2.59928,0l0,2.59928l-2.59928,0l0,-2.59928zm1.84201,1.84201l0,-1.08473l-1.08473,0l0,1.08473l1.08473,0zm-5.19855,0l0,-1.08473l-1.08473,0l0,1.08473l1.08473,0zm-1.84201,3.35654l2.59928,0l0,2.59928l-2.59928,0l0,-2.59928zm8.53463,0.34793l0,-1.08473l-1.0745,0l0,1.08473l1.0745,0zm-2.98815,0l0,-1.08473l-1.09496,0l0,1.08473l1.09496,0zm-3.70448,1.49407l0,-1.08473l-1.08473,0l0,1.08473l1.08473,0zm5.19855,0l0,-1.08473l-1.08473,0l0,1.08473l1.08473,0zm-1.49407,1.49407l0,-1.07451l-1.09496,0l0,1.07451l1.09496,0zm2.98815,0l0,-1.07451l-1.0745,0l0,1.07451l1.0745,0zm0.13303,-5.49531l0,-3.46912c0,-0.24559 -0.19444,-0.4298 -0.44004,-0.4298l-3.45888,0c-0.25583,0 -0.44004,0.1842 -0.44004,0.4298l0,3.46912c0,0.24559 0.1842,0.4298 0.44004,0.4298l3.45888,0c0.24559,0 0.44004,-0.1842 0.44004,-0.4298zm-8.66767,-3.03932l2.59928,0l0,2.59928l-2.59928,0l0,-2.59928zm3.46912,8.22763l0,-3.45888c0,-0.25583 -0.1842,-0.44004 -0.4298,-0.44004l-3.46912,0c-0.24559,0 -0.4298,0.1842 -0.4298,0.44004l0,3.45888c0,0.24559 0.1842,0.44004 0.4298,0.44004l3.46912,0c0.24559,0 0.4298,-0.19444 0.4298,-0.44004z"/>
+    </g>
+</svg>
```

Comment:
Raname file to `icon-qrcode.svg` to match other icon names.

Reply:
good catch 👍